src/manual/tasks/workertypes.md

```diff
@@ -0,0 +1,17 @@
+---
+filename: tasks/workertypes.md
+title: Worker Types
+order: 20
+---
+
+Workers execute tasks, but there are many types of workers. A task specifies a
+single "worker type" by which it should be executed, using the two-part
+namespace `<provisionerId>/<workerType>`.  Workers with the same worker type
+all consume tasks from a single queue, as [described
+later](/manual/task-execution/queues), and as such are interchangeable and have
+identical configurations.
```

Comment:
I'll adjust the next section (on priorities) to say "when several tasks are pending in the same queue, those with the highest priority.." which implies that different priorities are in the same queue (that they are in different Azure queues is an implementation detail)